util/design/data/secded_cfg.hjson

```diff
@@ -10,5 +10,6 @@
     {k: 16,   m: 6,  code_type: "hamming"},
     {k: 32,   m: 7,  code_type: "hamming"},
     {k: 64,   m: 8,  code_type: "hamming"},
-  ]
+  ],
+  seed: 1592631616
```

Comment:
would you mind adding a comment here like with `top_earlgrey.hjson` to ask user to change this explicitly when they do want to run with a different seed?

Reply:
yup, definitely.